crates/miden-lib/src/account/faucets/mod.rs

```diff
@@ -197,10 +227,12 @@ pub fn create_basic_fungible_faucet(
     account_storage_mode: AccountStorageMode,
     auth_scheme: AuthScheme,
 ) -> Result<(Account, Word), FungibleFaucetError> {
-    // Atm we only have RpoFalcon512 as authentication scheme and this is also the default in the
-    // faucet contract.
-    let auth_component: RpoFalcon512 = match auth_scheme {
-        AuthScheme::RpoFalcon512 { pub_key } => RpoFalcon512::new(pub_key),
+    let distribute_proc_root = BasicFungibleFaucet::distribute_digest();
+
+    let auth_component: RpoFalcon512ProcedureACL = match auth_scheme {
```

Comment:
Adding this component changes the storage layout of the overall account resulting from this function call, so we should update the documented storage layout.

Reply:
done [docs: update storage layout of faucet](https://github.com/0xMiden/miden-base/pull/1531/commits/d18ae0b33e06e4b2bd0d0a0f3880f7e743d38128)